### ISSUE TITLE
File configuration fails with input & output

### ISSUE THREAD

**Issue by ccsaposs** [posthtml/posthtml-cli#313]
Hi, I'm running into issues using the CLI when I specify the input or output with a configuration file.

For example, this command & configuration works:
`posthtml src/views/*.html -o public -c posthtml.json`

posthtml.json
```
{
  "plugins": {
    "posthtml-modules": {
      "root": "./src/views"
    }
  }
}
```

But this command & configuration fails (no files are outputted):
`posthtml -c posthtml.json`

posthtml.json
```
{
  "input": "src/views/*.html",
  "output": "public",
  "plugins": {
    "posthtml-modules": {
      "root": "./src/views"
    }
  }
}
```

Am I doing something wrong?

Thank you for your time.

**Comment by Scrum** [comment]
@ccsaposs Hi, thanks for your feedback, there are indeed a couple of issues with merging configurations.

**Comment by giuseppeg** [comment]
@Scrum not sure if this is related but one of the recent releases broke `input` when passed as CLI argument. It seems that now [`config.input`](https://github.com/posthtml/posthtml-cli/blob/27d39bac8fe9b13a760d63992b63e9721fb235eb/src/cli.js#L108) is an array of absolute paths and it doesn't work with fast glob.

**Comment by Scrum** [comment]
@giuseppeg @ccsaposs Yes, there was a massive degradation of functionality according to the configuration resolution, fix it in #314

**Comment by giuseppeg** [comment]
@Scrum no worries! Thank you so much for fixing these quickly - by the way while I appreciate to get the quick fix, with my bug reports I don't want to rush and make pressure on you. For what concerns me it is ok if it takes a little more time to get the fix as I know that many of us do open source in our free time.

**Comment by giuseppeg** [comment]
@Scrum I tried 0.7.3 and the cli doesn't work yet. Here is how I use it:

```
posthtml "some/inputFolder/**/*.html" "!**/some/outFolder/**" -r some/inputFolder -a -o some/outFolder
```
In order for fast-glob to grab those files here you shouldn't resolve the path to absolute - you should remove the following line 

https://github.com/posthtml/posthtml-cli/blob/461615213ccd1bb66817582f9f42572d34946cfb/src/cfg-resolve.js#L32

**Comment by Scrum** [comment]
> no worries! Thank you so much for fixing these quickly - by the way while I appreciate to get the quick fix, with my bug reports I don't want to rush and make pressure on you. For what concerns me it is ok if it takes a little more time to get the fix as I know that many of us do open source in our free time.

You do not create pressure, I know not by hearsay what it is to wait for an answer or a decision on GitHub and this does not give me joy, I would not want the products of the organization to suffer from this.

Everyone is yours issue this is a small step for our organization in its development, do not hesitate to write or talk about problems, we will always find time to quickly solve your problem

**Comment by ccsaposs** [comment]
@Scrum Thank you so much for the quick fix!  We've got 0.7.3 up and running & it's working great.  We really appreciate your quick response and the work that you've done on this project.
